postprocessing/pp-opus.c

```diff
@@ -72,31 +78,147 @@ int janus_pp_opus_process(FILE *file, janus_pp_frame_packet *list, int *working)
 	int bytes = 0, len = 0, steps = 0, last_seq = 0;
 	uint64_t pos = 0, nextPos = 0;
 	uint8_t *buffer = g_malloc0(1500);
+
+#ifdef HAVE_LIBOPUS
+	int error = 0;
+	OpusDecoder *decoder = NULL;
+	OpusEncoder *encoder = NULL;
+	opus_int16 *pcm_data = NULL, *out_buffer = NULL;
+	int max_frame_size = 120;
+	int channels = 1;
+	int sampling_rate = 8000;
+	gsize buflen;
+	if(use_fec) {
+		/* Check content paramters (mono/stereo, bandwidth) */
+		while(*working && tmp != NULL) {
+			len = 0;
+			/* RTP payload */
+			offset = tmp->offset+12+tmp->skip;
```

Comment:
12 is just the default rtp header length, are extensions stripped elsewhere?

Reply:
`tmp->skip` should contain the additional offset